ci/gpu/build.sh

```diff
@@ -64,7 +62,8 @@ conda install -c pytorch -c gwerbin \
     "s3fs" \
     "ipython" \
     "matplotlib" \
-    "nbconvert"
+    "nbconvert" \
+    "faker"
```

Comment:
Is `faker` just a test dependency? If not, if also needs to be added to the conda recipe.

Reply:
yes, it's just a test dependency.